src/process_utility.c

```diff
@@ -1026,6 +1129,13 @@ process_rename(Node *parsetree)
 	}
 
 	ts_cache_release(hcache);
+
+	/* In case if command modifies a hypertable meta-data we need to
+	 * process it first here to sync up, before passing to tsl hook */
```

Comment:
please make the comment. `In case the command modifies the relation's name we need to process the update here so that the relation's name and our catalog metadata is in-sync before the tsl hook is called`